Makefile

```diff
@@ -1,8 +1,9 @@
 NODE_ZERO_IP ?= 192.168.122.2
+API_VIP ?= 192.168.122.10
 
 .PHONY: iso
 iso: clean
-	go run cmd/main.go -node-zero-ip $(NODE_ZERO_IP)
+	go run cmd/main.go -node-zero-ip $(NODE_ZERO_IP) -apivip $(API_VIP)
```

Comment:
I also think it is better to read the api_vip from the manifests instead of from CLI. Thoughts?

Reply:
I agree that its better to read the api-vip from the manifests to have it defined in only one place.

Reply:
Done